src/org/traccar/api/BaseResource.java

```diff
@@ -29,4 +35,8 @@ protected long getUserId() {
         }
         return 0;
     }
+
+    protected void linkNewEntity(BaseModel entity) throws SQLException {
```

Comment:
I had this comment before, but why do we need this method? It just calls one other method. I think we should remove it. It would make it easier to follow the code.